implants/lib/eldritch/src/runtime/eval.rs

```diff
@@ -142,6 +142,16 @@ pub struct Runtime {
     rx: Receiver<Message>,
 }
 
+#[starlark_module]
+fn error_handler(builder: &mut GlobalsBuilder) {
+    #[allow(unused_variables)]
+    fn eprint(starlark_eval: &mut Evaluator<'v, '_>, message: String) -> anyhow::Result<NoneType> {
+        let env = crate::runtime::Environment::from_extra(starlark_eval.extra)?;
+        eprint_impl::eprint(env, message)?;
```

Comment:
Love that we pass env here, greatly demonstrates the separation between eprint (starlark) and eprint_impl (native rust)